src/components/ecl-spinner/ecl-spinner.stories.ts

```diff
@@ -1,7 +1,7 @@
 const getArgs = () => {
```

Comment:
overlay controls seems not to work the same as on v5 dev. Also a size control would be also nice.